app/domain/errors.rb

```diff
@@ -171,6 +171,11 @@ module AuthnOidc
         code: "CONJ00013E"
       )
 
+      ServiceIdMissing = ::Util::TrackableErrorClass.new(
+        msg:  "Service id is required when authenticating with authn-oidc",
+        code: "CONJ00074E"
+      )
```

Comment:
Looks like you got race-conditioned out of this `code` by a PR which landed yesterday:

https://github.com/cyberark/conjur/blob/master/app/domain/errors.rb#L28